version/version.go

```diff
@@ -8,7 +8,7 @@ var mainpkg = "github.com/distribution/distribution/v3"
 // the latest release tag by hand, always suffixed by "+unknown". During
 // build, it will be replaced by the actual version. The value here will be
 // used if the registry is run after a go get based install.
-var version = "v3.0.0-rc.2.m+unknown"
+var version = "v3.0.0-rc.3.m+unknown"
```

Comment:
Looks like the version has a `.m` (modified / dirty state) marker; I suspect that was a slip-up in https://github.com/distribution/distribution/commit/a74cacff04e76ec91230ed407a10d50f64c028ef#diff-f6bf0ae0d21c9408c6624174da11dafb369ce306f2cdcea704285d8acc87e19eL11-R11 that got copied through

Reply:
Fixed